Lib/test/test_inspect.py

```diff
@@ -107,6 +107,10 @@ def istest(self, predicate, exp):
                 continue
             self.assertFalse(other(obj), 'not %s(%s)' % (other.__name__, exp))
 
+    def test___all__(self):
+        self.assertGreater(len(inspect.__all__), 0)
+        self.assertIn("signature", inspect.__all__)
+        
```

Comment:
Have a look at test.support.check__all__

Reply:
That cannot be used as this module defines k as global

Reply:
Can you specify that as not_exported?
https://github.com/python/cpython/blob/a6c3b0faa1d55e36539caf19bd3bcf1dea12df84/Lib/test/support/__init__.py#L1496

Reply:
done